meetings/2024-11-20.md

```diff
@@ -0,0 +1,47 @@
+# W3C Solid Community Group: Weekly
+
+* Date: 2024-11-20T14:00:00Z
+* Call: https://meet.jit.si/solid-cg
+* Chat: https://matrix.to/#/#solid_specification:gitter.im
+* Repository: https://github.com/solid/specification
+* Status: Draft
+
+
+## Present
+* Hadrian Zbarcea
+* [name]
+
+---
+
+## Announcements
+
+
+### Meeting Guidelines
+* [W3C Solid Community Group Calendar](https://www.w3.org/groups/cg/solid/calendar).
+* [W3C Solid Community Group Meeting Guidelines](https://github.com/w3c-cg/solid/blob/main/meetings/README.md).
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Conduct](https://www.w3.org/policies/code-of-conduct/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+### Scribes
+* [name]
+
+### Introductions
+* [name]
+
+
+## Topics
+
+### Election of new co-chair
```

Comment:
Would it be possible for the current chairs to come up with a more sympathetic and appreciative title for the agenda topic, considering a fellow co-chair stepping down, and acknowledging their leadership and efforts on the CG?

Since this is not even in the Announcements section of this agenda, even a "boring" title  repeating the email Subject would be more appropriate IMHO:

```suggestion
### Stepping down as co-chair of the Solid CG
```

Celebrating their service to the community should be "the" topic.

And then perhaps follow-up with "next steps" which you're surely more keen about?

Reply:
Makes sense! I went ahead and inserted the 'thanks' topic before the 'next steps' topic in https://hackmd.io/b4FdKG4lTYO2h74U933YHA